Dashboard/app/js/lib/views/data/question-answer-view.js

```diff
@@ -71,8 +77,12 @@ FLOW.QuestionAnswerView = Ember.View.extend({
 
   isNotEditable: function(){
     var type = this.get('questionType');
-    return (type == 'GEO' || type == 'PHOTO' || type == 'VIDEO' || type == 'GEOSHAPE');
-  }.property('this.questionType'),
+    return (type == 'GEO' || type == 'PHOTO' || type == 'VIDEO' || type == 'GEOSHAPE' || !this.get('isEditable'));
+  }.property('this.questionType,this.isEditable'),
+
+  isEditable: function () {
+    return FLOW.permControl.canEditResponses(this.get('form'));
+  }.property('this.form'),
```

Comment:
Without getting into the internal details (Bowels of Ember ™) and implementation, this 2 functions feel somehow weird. Wouldn't a single `isEditable()` suffice? As a maintainer, I would fail to see why I should use one or the other.